tools/browsers-lists/extended-debugging.mjs

```diff
@@ -0,0 +1,19 @@
+import browsersAll from './browsers-all.json' assert { type: "json" }
+
+export default function browserSupportsExtendedDebugging ({ browserName, browserVersion, version }) {
+  if (!['chrome', 'firefox'].includes(browserName)) {
+    return false
+  }
+
+  if (browserVersion === 'latest') {
+    return true
+  }
+
+  if (browserName === 'firefox' && (Number(browserVersion) >= 53 || Number(version) >= 53)) {
+    return true
+  }
+
+  const latestChrome = browsersAll.chrome
+    .reduce((aggregator, sauceBrowser) => aggregator = Math.max(Number(sauceBrowser.version), Number(sauceBrowser.browserVersion), aggregator), 0)
+  return (Number(browserVersion) || Number(version)) >= latestChrome - 2
+}
```

Comment:
Side comment: it may be faster to just sift thru `browsers-supported` instead for what would practically be the same result. Also, `firefox` < 53 would fall down here though it would realistically never fulfill this condition, of course.

Reply:
Good point.